templates/index.html

```diff
@@ -11,7 +11,31 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li style="list-style-type: none;">
+            <form action="/remove-item" method="POST" style="display: inline;">
```

Comment:
If you wanted to you could use "ur_for()" here (but not important)

Reply:
Done it!